include/gatekeeper_net.h

```diff
@@ -19,7 +19,19 @@
 #ifndef _GATEKEEPER_NET_H_
 #define _GATEKEEPER_NET_H_
 
-int gatekeeper_init_network(void);
+#include <stdint.h>
+
+/* Configuration for the Network. */
+struct net_config {
+	uint16_t		num_rx_queues;
+	uint16_t		num_tx_queues;
+	uint32_t		num_ports;
```

Comment:
Write the following comment between `num_tx_queues` and `num_ports`:

/* The fields below are for internal use. Configuration files should not refer to them. */